lib/linux_admin/network_interface.rb

```diff
@@ -112,7 +114,11 @@ def netmask6(scope = :global)
     #
     # @return [String] IPv4 gateway address
     def gateway
-      @network_conf[:gateway]
+      @network_conf[:gateway4]
+    end
+
```

Comment:
Ha found it. `ip -6 route show dev #{name}` I can do it in follow-up pr.

Or we can wait until it is broken for somebody. We are still using old network device naming `eth0` and not the dynamic one. As a side effect the ethX can change in order cause a disruption.

So my answer to anyone with custom setup would be: don't use appliance_console, use what you have learned in RHCSA. :)